invenio_app_rdm/communities_ui/views/communities.py

```diff
@@ -89,3 +92,25 @@ def communities_home(pid_value, community, community_ui):
             permissions=permissions,
             records=records_ui,
         )
+
+
+@pass_community(serialize=True)
+def community_static_page(pid_value, community, community_ui):
+    """Community static page."""
+    permissions = community.has_permissions_to(
```

Comment:
question: does it make sense to check for permissions and throw a `PermissionDeniedError`? Like above in the `communities_home` view function.